### DISCUSSION TITLE
Distance around claim if mining to bedrock?

### DISCUSSION THREAD

**Discussion by xSepticSidx** [GriefPrevention/GriefPrevention#1873] [Q&A] [answered]
HI, As I understand it, on the server I am on the claim depth is set to the default 5.

In order to prevent others tunneling under my claims I intend to mine down to bedrock, however I would like to know how far around me I have to go?
For instance, do I need to mine down in 1 spot in a claim only and that affects all of the claim or is there a radius around the digging down player to consider?

**Comment by Jikoo** [comment, marked as answer]
You only need to affect one block inside the claim, however, digging down alone is not enough. You need to place a block that is clearly player-made (like a crafting table, a chest, anything unnatural) before the claim will expand downwards.
Alternately, you can create the claim at bedrock and resize it later.

The server must also have set `GriefPrevention.Claims.MaximumDepth` to a value at or below the Y level you're trying to get your claim to encompass.

**Reply by xSepticSidx** [reply]
Thank you.
